source/agora/test/ManyValidators.d

```diff
@@ -80,10 +80,10 @@ void manyValidators (size_t validators)
     network.assertSameBlocks(Height(GenesisValidatorCycle + 1));
```

Comment:
Commit messages should explain why a change was made and not just what was changed. The last two commits don't explain the reasoning behind the changes.